apps/typegpu-docs/src/examples/tests/tgsl-parsing-test/index.html

```diff
@@ -1 +1 @@
-<div class="result">Wait for the tests to finish running...</div>
+<div class="result">Wait for the tests to finish running...</div>;
```

Comment:
The formatter sometimes adds a semicolon at the end for some reason
```suggestion
<div class="result">Wait for the tests to finish running...</div>
```